src/Applications/GEOSdas_App/fvsetup

```diff
@@ -579,21 +579,22 @@ $ENV{"PATH"} = $FindBin::Bin .":$ENV{PATH}";
   die "Failed to fix GOCART levels-referencing $!" if ( $rc );
 
 # Call LDAS setup script.
-#  Provide LDAS config input files in $ldassetup/: det_exeinp.txt, det_batinp.txt. 
-#  For Hyb-4dEnVar, do same for ens_exeinp.txt, ens_batinp.txt. 
+#  Provide LDAS config input files in $ldassetup/: ldas_exeinp.txt, ldas_batinp.txt. 
 #  See GEOSldas/README.md for generating sample config input files.
 # ------------------------------------------------------------------------
     print "\n checkprintout  ldas_ana == 1 ($ldas_ana)\n";
     if ($ldas_ana == 1 ) {
-        $cmd = "$fvbin/ldas_setup setup $fvhome/lana --account $gid --nymdb $nymdb --nhmsb $nhmsb $ldassetup/det_exeinp.txt $ldassetup/det_batinp.txt";
+        $cmd = "$fvbin/ldas_setup setup $fvhome/lana --account $gid --nymdb $nymdb --nhmsb $nhmsb --ladas_cpl 1 --agcm_res $res --rstdir $ldasics $ldassetup/ldas_exeinp.txt $ldassetup/ldas_batinp.txt";
```

Comment:
The variable $res seems to exist in the desired format elsewhere in fvsetup as $agcm_im4.  Could we use $agcm_im4 here to avoid the string modifications inside ldas_setup? 
There is also another variable, $bcsresa, that seems to hold the same info. 
Also, what we are really interested in here is the BCS info.  The BCS info consists of the AGCM grid & resolution as well as the version.  From $res alone we can't identify the bcs that need to be used in LDAS.

Reply:
will take a closer look at BCs version.